synapse/tests/test_lib_stormlib_storm.py

```diff
@@ -36,3 +36,28 @@ async def test_lib_stormlib_storm(self):
 
             with self.raises(s_exc.IsReadOnly):
                 await core.callStorm('$lib.storm.eval( "{$lib.auth.users.add(readonly)}" )', opts={'readonly': True})
+
+            with self.getLoggerStream('synapse') as stream:
```

Comment:
can we get a specific logger here instead of the `synapse` logger?

Reply:
I used `synapse` because I wanted output from both `synapse.storm` and `synapse.lib.stormlib.storm` and thought this would be easier.

Reply:
Although, I'm realizing now that I should have used `<core>._logStormQuery()` so updating with that.